public/js/graph/react_node.js.jsx

```diff
@@ -0,0 +1,109 @@
+function renderReactGraph() {
+    React.render(
+        <ReactSVG width={1195} height={650}/>,
+        document.getElementById('react-graph')
+    );
+}
+
+function getAttributes(svgAttributes) {
+    var attrs = [];
+    //Traversing a NodeNamedMap type
+    Array.prototype.slice.call(svgAttributes).forEach(function(item) {
+        attrs[item.name] = item.value;
+    });
+    return attrs;
+}
+
+function getStyles(styles_strings) {
+    var styles = {};
+    if (!styles_strings) {
+        return styles;
+    }
+    styles_strings.split(";").map(function(key, value){
+        if (key) {
+            styles[key.substring(0, key.indexOf(':'))] = key.substring(key.indexOf(':')+1);
+        }
+    });
+    return styles;
+}
+
+var ReactSVG = React.createClass({
+    render: function() {
+        return (
+            <svg width={this.props.width} height={this.props.height}>
+                {/*<ReactRegions/>*/}
+                <ReactNodes/>
+                {/*<ReactRegions/>*/}
+                {/*<ReactBools/>*/}
+                {/*<ReactEdges/>*/}
+                {/*<ReactRegionLabels/>*/}
+            </svg>
+        );
+    }
+});
+
+var ReactNodes = React.createClass({
+    getInitialState: function(){
+        return {
+            nodes_list: []
+        };
+    },
+    
+    componentDidMount: function(){
+        var dict_list = [];
+        
+        $('.node').map(function(key, value) {
+            var entry = {};
+            entry["id"] = value.id;
+            entry["attributes"] = getAttributes(value.attributes);
+            //assume no styles here
```

Comment:
Is there any trade-off to assuming no styles here? Why is this being done? Is there any way to add in more helpful information to this comment?